lib/autosave.coffee

```diff
@@ -34,6 +35,7 @@ module.exports =
     return unless atom.config.get('autosave.enabled')
     return unless paneItem?.getURI?()?
     return unless paneItem?.isModified?()
+    return unless paneItem?.getPath()? and fs.isFileSync(paneItem.getPath())
```

Comment:
What happens here if a pane item doesn't implement `::getPath()`? Won't it throw a `TypeError: undefined is not a function`? I'd add another existential operator guard here like the above lines just to be safe:

```coffee
paneItem?.getPath?()?
```